proxy/InkAPI.cc

```diff
@@ -8256,6 +8265,17 @@ TSHttpTxnConfigStringSet(TSHttpTxn txnp, TSOverridableConfigKey conf, const char
     if (value && length > 0) {
       s->t_state.txn_conf->client_cert_filepath = const_cast<char *>(value);
     }
+  case TS_CONFIG_HTTP_RESPONSE_VIA_TRANSPORT:
+    data._s.setView(value, length);
+    if (ViaTransportVerbosityConversion(s->t_state.txn_conf->response_via_transport, "proxy.config.http.response_via_tranport",
+                                        RECD_STRING, data)) {
+    }
+    break;
+  case TS_CONFIG_HTTP_REQUEST_VIA_TRANSPORT:
+    data._s.setView(value, length);
+    if (ViaTransportVerbosityConversion(s->t_state.txn_conf->request_via_transport, "proxy.config.http.request_via_tranport",
```

Comment:
I think @zwoop rise up the right questions, the via config in records looks very usable, you can add more verbose syntax if you like to 4 5 6 ... I think that is the way we need to go. adding another complex is sure a very not so nice feature, as via complex is way too much already. introduce a new Rec Type is for this only feature is another must to kill idea.